### PR TITLE
Set displayUrl to be the unadapted url (closes #24)

### LINKED ISSUE #24: Adaptor tiles should show original URL
We shouldn't expose the adaptor URL to the end user in any way. If they follow an adaptor tile it should take them to the original discovered link.

### PR DESCRIPTION
**Pull request by wilsonpage**

### CODE REVIEW

**Comment by wilsonpage** [comment]
@arcturus r?

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "lint": "eslint .",
     "start": "node index.js",
-    "pretest": "npm run lint",
+    "posttest": "npm run lint",
```
Comment (wilsonpage) [inline package.json:9]: This was preventing us putting console.logs in unit-tests

File: test/api.js [hunk 2]
```diff
@@ -27,14 +23,6 @@ describe('api', () => {
       .end(done);
   });
 
-  it('should perform basic analysis', (done) => {
-    request(app)
-      .post('/metadata')
-      .send(basicSites)
-      .expect(200)
-      .end(done);
-  });
```
Comment (wilsonpage) [inline test/api.js:36]: This test wasn't testing anything and was hitting the network.

**Comment by arcturus** [comment]
r+ shiiiip it!   🐫

**Pull request merged**